javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,90 @@
+// Shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function compareLength(arr) {
+  return arr.reduce((x, y) => (x.length <= y.length ? x : y));
+}
+console.log("The shortest word in array is " + compareLength(danishWords));
+// returns 'ø' << Added console log here instead of consolelogging inside func
+
+//Danish letter amount
+function countLetters(string) {
+  console.log(string);
```

Comment:
One thing that's worth thinking about here is input normalization: can we always count on the input characters being lowercase? Probably not, so it's a good idea to add a `string.toLowerCase()` somewhere here, so that you're 100% certain of what you're dealing with later